src/rpc/client.cpp

```diff
@@ -105,6 +105,10 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "getaddressdeltas", 0},
     { "getaddressutxos", 0},
     { "getaddressmempool", 0},
+    { "getblockhashes", 0},
+    { "getblockhashes", 1},
+    { "getblockhashes", 2},
+    { "getblockdeltas", 0},
```

Comment:
The `getblockdeltas` change should have been in #3953. I'll have to go back and figure out why I missed it in that review.

Reply:
Yes, sorry I missed that too; the reason I didn't catch it is because I added argument-processing tests to `src/test/rpc_tests.cpp` in the current PR; #3953 didn't include those tests for this RPC (I had missed it).